CONTRIBUTING.md

```diff
@@ -1,63 +1,64 @@
 How to report a bug
 -------------------
-- Use the search function to see if someone else has already reported the bug. If an issue has already been opened, see if you can add any useful information to it.
-- Otherwise, create a new issue on the "issues" page
-- Describe the problem as detailed as possible.
-- Some bugs only occur on certain devices or versions of Android. Please add information about your device and the version of Android that is running on it (you can look these up under `Settings → About Phone`) as well as which version of AntennaPod you are using.
-- If the bug only seems to occur with a certain podcast, make sure to include the URL of that podcast.
+- Use the search function to see if someone else has already submitted the same bug report.
+- Try to describe the problem with as much detail as possible.
+- Some bugs may only occur on certain devices or versions of Android. Please add information about your device and the version of Android that is running on it (you can look these up under `Settings → About Phone`), as well as which version of AntennaPod you are using.
+- If the bug only seems to occur with a certain podcast, please include the URL of that podcast.
 - If possible, add instructions on how to reproduce the bug.
-- If possible, add a logfile to your post. This is especially useful if the bug makes the application crash. You can create logfiles with an app like `aLogcat`. Just launch `alogcat`, then start AntennaPod and reproduce the bug. aLogcat should now display information about the bug when you start it.
-- Usually, you can make a screenshot of your smartphone by pressing [Power] + [Volume down] for a few seconds
-- Please use the following **template**: 
+- If possible, add a logfile to your post. This is especially useful if the bug makes the application crash. You can create logfiles with apps like `aLogcat`. Just launch `alogcat`, then start AntennaPod and reproduce the bug. aLogcat should now display information about the bug when you start it. AntennaPod also has an `export logs` feature.
+- Usually, you can take a screenshot of your smartphone by pressing *Power* + *Volume down* for a few seconds.
+- Please use the following **[template](.github/ISSUE_TEMPLATE/bug_report.md)**.
 
-**App version**: 1.x (from Google Play/F-Store/Custom build)
+**App version**: (state whether from Google Play/F-Droid/Custom built APK)
```

Comment:
Sorry, what I meant with `Please only delete the part about the template. Everything else is useful, I think.` was: Please remove the `App version ... Stack trace` part (as it is just copied from the template) but keep the explanation above.